docs/nlu/components.rst

```diff
@@ -116,6 +116,15 @@ SpacyFeaturizer
 :Requires: :ref:`SpacyNLP`
 :Description:
     Creates feature for intent classification using the spacy featurizer.
+    Optionally adds word vectors for each ``token`` to ``ner_features``, which can be referenced in ``CRFEntityExtractor``
+:Configuration:
+
+    .. code-block:: yaml
+
+        pipeline:
+        - name: "SpacyFeaturizer"
+          # Whether to add word vectors to ``ner_features``
+          ner_feature_vectors: False
```

Comment:
So, if `ner_feature_vectors: False` then `ner_features` are set or not? `False` is kind of unintuitive. I would expect setting `ner_feature_vectors` to `True` would result in adding them.

Reply:
And maybe also mention the default value.

Reply:
Oh I thought I had to put the default value here. It works the way you originally thought, but it's `False` by default. I'll update the docs to say `True` but note that it's not the default